### PR TITLE
docs(quickstart): use app.router

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: docs/source/zh-cn/intro/quickstart.md [hunk 1]
```diff
@@ -401,8 +403,7 @@ module.exports = SomeService;
 
 ```js
 // test/app/middleware/robot.test.js
-const assert = require('assert');
-const mock = require('egg-mock');
+const { app, mock, assert } = require('egg-mock/bootstrap');
 
 describe('test/app/middleware/robot.test.js', () => {
   let app;
```
Comment (dead-horse) [inline docs/source/zh-cn/intro/quickstart.md]: 这里要去掉 app 了

Comment (dead-horse) [reply docs/source/zh-cn/intro/quickstart.md]: 其他 +1

Comment (atian25) [reply docs/source/zh-cn/intro/quickstart.md]: 改了

File: docs/source/en/intro/quickstart.md [hunk 2]
```diff
@@ -429,11 +430,9 @@ All the test files should place at `{app_root}/test/**/*.test.js`.
...
 ```js
 // test/app/middleware/robot.test.js
-const assert = require('assert');
-const mock = require('egg-mock');
+const { app, mock, assert } = require('egg-mock/bootstrap');
 
 describe('test/app/middleware/robot.test.js', () => {
-  let app;
   before(() => {
     app = mock.app();
```
Comment (dead-horse) [inline docs/source/en/intro/quickstart.md]: 这里也不需要了啊，不需要 before 了

Comment (atian25) [reply docs/source/en/intro/quickstart.md]: fixed, after 也干掉了

**Pull request merged**